.github/workflows/test.yml

```diff
@@ -8,8 +8,7 @@ on:
 
 jobs:
   test:
-    name: Elixir ${{matrix.elixir}} / OTP ${{matrix.otp}}
```

Comment:
![image](https://github.com/absinthe-graphql/dataloader/assets/134518/35929d8f-4fe9-43df-9a20-33ae7574c8ed)

We want to show the `lint` flag in job name.